### PR TITLE
Adding tests for parsing one line functions/ Fixing parser bug.

### PR DESCRIPTION
**Pull request by theyuvalraz**

### CODE REVIEW

File: src/tests/parser_test.py [hunk 1]
```diff
@@ -888,6 +888,12 @@ def test():
         def test():
             pass
     """),
+    CodeSnippet("""\
+        '''Test this'''
+
+        @property
```
Comment (samj1912) [inline src/tests/parser_test.py:894]: Can you also add some tests for multiple decorators applied to single line function?

**Review by samj1912 — CHANGES_REQUESTED** [review]

File: src/tests/parser_test.py (hunk 1 above)
Comment (theyuvalraz) [reply src/tests/parser_test.py:886]: Sure, Added this scenario in the parameterised test.
without adding a new test i can't test that both decorators are recorded, will add it later if you think it's needed.

File: src/tests/test_cases/test.py [hunk 2]
```diff
@@ -337,6 +337,16 @@ def oneliner_d102(): return
 def oneliner_withdoc(): """One liner"""
 
 
+def ignored_decorator(func):   # noqa: D400,D401,D415
+    """Runs something"""
+    func()
+    pass
+
+
+@ignored_decorator
```
Comment (samj1912) [inline src/tests/test_cases/test.py:346]: Can you please add a test case next to this one with a single line definition similar to this one that doesn't have an ignored decorator and raises errors and another one that has a single line definition with a docstring that doesn't raise any errors to check that everything is working as expected in this context?

Comment (theyuvalraz) [reply src/tests/test_cases/test.py:352]: Good idea :)
Are the tests I added meet the criteria?

**Review by samj1912 — APPROVED** [review]
lgtm, thanks for the PR!

**Comment by samj1912** [comment]
@theyuvalraz I see that we forgot to update the release notes for this PR. Can you please add a separate PR that updates them and references this one?

**Pull request merged**